src/material/stepper/_stepper-theme.scss

```diff
@@ -100,8 +120,17 @@
     background-color: theming.get-color-from-palette($background, card);
   }
 
+  .mat-horizontal-content-container, .mat-vertical-content-container {
+    @include a11y.high-contrast(active, off) {
+      outline: solid 1px;
+    }
+  }
+
   .mat-stepper-vertical-line::before {
     border-left-color: theming.get-color-from-palette($foreground, divider);
+    @include a11y.high-contrast(active, off) {
```

Comment:
Why do we need to remove this border?

Reply:
I removed it at first because I thought there was too many lines with the outlines over the buttons in the step header. Although looking at it again it looks fine so I added it back in.